src/UserService.Data/UserRepository.cs

```diff
@@ -64,5 +65,19 @@ public DbUser GetUserByEmail(string userEmail)
 
             return dbUser;
         }
+
+        public IEnumerable<DbUser> GetAllUsers(int skipCount, int takeCount, string nameFilter)
```

Comment:
Rename nameFilter -> userFullName

Reply:
![image](https://user-images.githubusercontent.com/59228285/96621596-7a2be180-1311-11eb-943e-095ad567af52.png)
That's exactly a filter, not the whole full name

Reply:
it is not clear what the filter is, the nameFilter does not make it clear to me what the variable means. nameFilter -> userNameFilter

Reply:
Agreed. Fixed